packages/base/src/3dview/mainview.tsx

```diff
@@ -362,29 +362,29 @@ export class MainView extends React.Component<IProps, IStates> {
 
       this._transformControls.enabled = false;
       this._transformControls.visible = false;
+      this._createViewHelper();
+    }
+  };
 
-      // ViewHelper setup
-      this._viewHelper = new ViewHelper(
-        this._camera,
-        this._renderer.domElement
-      );
-      this._viewHelper.center = this._controls.target;
+  private _createViewHelper() {
+    this._viewHelper = new ViewHelper(this._camera, this._renderer.domElement);
+    this._viewHelper.center = this._controls.target;
 
-      const viewHelperDiv = document.createElement('div');
-      viewHelperDiv.id = 'viewHelper';
-      viewHelperDiv.style.position = 'absolute';
-      viewHelperDiv.style.right = '0px';
-      viewHelperDiv.style.bottom = '0px';
-      viewHelperDiv.style.height = '128px';
-      viewHelperDiv.style.width = '128px';
+    const viewHelperDiv = document.createElement('div');
+    viewHelperDiv.style.position = 'absolute';
+    viewHelperDiv.style.right = '0px';
+    viewHelperDiv.style.bottom = '0px';
+    viewHelperDiv.style.height = '128px';
+    viewHelperDiv.style.width = '128px';
 
-      this.divRef.current.appendChild(viewHelperDiv);
+    this._viewHelperDiv = viewHelperDiv;
 
-      viewHelperDiv.addEventListener('pointerup', event =>
-        this._viewHelper.handleClick(event)
-      );
-    }
-  };
+    this.divRef.current?.appendChild(this._viewHelperDiv);
```

Comment:
Just a question, how are we removing the old one now?

Reply:
We're not removing it now as it was not necessary, just updating it with `updateCamera` works.

Reply:
What I mean is that you re-create a new `this._viewHelperDiv` in `updateCamera`, that you append to `this.divRef.current`.

Don't we need to remove the old `this._viewHelperDiv` that was created before?

Reply:
<img width="492" alt="image" src="https://github.com/user-attachments/assets/30f232c3-57c7-4baa-b1d4-af01b0786a48">

Yes you're right, I just verified. Sorry that I overlooked, I'll be omw to fix it.

Reply:
Done in [`4602e00`](https://github.com/jupytercad/JupyterCAD/pull/469/commits/4602e00cff6d62640a924ed800558c256c3c22fe)

Thank You for pointing this out :)